cmd/cape/cmd/delete.go

```diff
@@ -71,7 +80,13 @@ func doDelete(url string, functionID string, insecure bool, auth entities.Functi
 		if res.StatusCode == 404 {
 			return fmt.Errorf("function ID not found: %s", functionID)
 		}
-		return fmt.Errorf("expected 200, got server response code %d", res.StatusCode)
+
+		errMsg := ErrorMsg{}
```

Comment:
Use `var errMsg ErrorMsg` to create an empty struct

Reply:
happy to do this but do we have a style guide we're following or something else?